sdk/appconfiguration/Azure.Data.AppConfiguration/src/Generated/ConfigurationClientOptions.cs

```diff
@@ -13,8 +13,5 @@ namespace Azure.Data.AppConfiguration
     /// <summary> Client options for ConfigurationClient. </summary>
     public partial class ConfigurationClientOptions : ClientOptions
     {
-        private const ServiceVersion LatestVersion = ServiceVersion.V1_0;
-
-        internal string Version { get; }
     }
 }
```

Comment:
Generator is generating an empty ConfigurationClientOptions class which is not needed as there is a customized [ConfigurationClientOptions](https://github.com/Azure/azure-sdk-for-net/blob/832222abd0ac0281651d0cf8a804415300f78dd4/sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs) class available with all the required properties outside of the generated folder.

HLC generated code doesn't generate an empty ClientOptions class if there is a customized class available outside of the generated folder. For example: there is no ClientOptions class generated for TextAnalytics [here](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated).

Reply:
Tracking this issue here: https://github.com/Azure/autorest.csharp/issues/3048